rust/capture/src/limiters/redis.rs

```diff
@@ -185,15 +147,12 @@ mod tests {
             .zrangebyscore_ret("@posthog/quota-limits/events", vec![String::from("banana")]);
         let client = Arc::new(client);
 
-        let limiter = RedisLimiter::new(Duration::microseconds(1), client, None)
+        let limiter = RedisLimiter::new(Duration::seconds(1), client, None, QuotaResource::Events)
             .expect("Failed to create billing limiter");
+        tokio::time::sleep(std::time::Duration::from_millis(30)).await;
```

Comment:
i don't like having these sleeps in the test but i can't figure out a better way to get the update thread to actually do something